pkgs/by-name/mi/minhtml/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
Rust builder now supports finalAttrs-style lambdas, prefer them over rec whenever possible